src/engine/TransitivePathImpl.h

```diff
@@ -244,7 +246,7 @@ class TransitivePathImpl : public TransitivePathBase {
    */
   NodeGenerator transitiveHull(const T& edges, LocalVocab edgesVocab,
                                std::ranges::range auto startNodes,
-                               std::optional<Id> target) const {
+                               std::optional<Id> target, bool yieldOnce) const {
```

Comment:
Add to the docstring, that the  yieldOnce has to have the same value as in the using code, and that it is only used for optimizing the local vocabs.